pages/linux/service.md

```diff
@@ -18,3 +18,7 @@
 - List the status of all services:
 
 `service --status-all`
+
+- Restart network-manager service
```

Comment:
This example is covered with the first example

Reply:
Yes, however, this example is an **example of** the first example. I'm hoping, people who are looking at it for the first time will find it helpful.

Reply:
This is the kind of problem that  the `{{token_syntax}}` is supposed to solve. Here's what the existing page renders as:

![](https://user-images.githubusercontent.com/9929737/94963014-57b35f00-04ef-11eb-8599-ee86f4badd99.png)

Note the cyan text, that indicates that something can be changed.

I'm not convinced that duplicate example will help here - another principle that we follow here at tldr-pages is keeping pages concise.

Perhaps we could instead improve this page by changing `init_script` to `service_name`? Perhaps that would aid clarity without making the page unnecessarily verbose.

Reply:
The idea of swapping out `service_name` with `init_script` sounds good. We should also add the `service --status-all` example first instead of last, with the title `List the status of all services:` swapped out for `List the name and status of all services`. Understand that the kind of audience that uses tldr-pages is often the kind who is avoiding the un-natural reading ambiance of man pages.

Reply:
Sounds good, @Ray6464.

Reply:
Made some changes to the initial PR.